extensions/interactions/base.py

```diff
@@ -89,6 +89,11 @@ class BaseInteraction(object):
     # containing this interaction. These should correspond to names of files in
     # feconf.DEPENDENCIES_TEMPLATES_DIR. Overridden in subclasses.
     _dependency_ids = []
+    # Additional JS library dependencies that should be loaded in pages
```

Comment:
Ah, I agree. Just one question, should I keep directory structure as it is or modify that too? If we plan to keep it as it is then we will have to check two directories instead of one.